services/meta/store_fsm.go

```diff
@@ -146,8 +146,13 @@ func (fsm *storeFSM) applyCreateNodeCommand(cmd *internal.Command, peers []strin
 		}
 	}
 
-	if err := other.CreateDataNode(v.GetHost(), v.GetHost()); err != nil {
-		return err
+	// Get the only meta node
+	if len(other.MetaNodes) == 1 {
```

Comment:
Will this break starting up a brand new server?  How does the `DataNode` get created in that case?